compendium/DeclarativeServices/test/TestManifestWithInvalidConfiguration.cpp

```diff
@@ -0,0 +1,129 @@
+  cppmicroservices::Bundle testBundle = StartTestBundle("TestBundleDSCA14");
+
+  // Use DS runtime service to validate the component state.
+  scr::dto::ComponentDescriptionDTO compDescDTO;
+  auto compConfigs =
+    GetComponentConfigs(testBundle, componentName, compDescDTO);
+
+  EXPECT_EQ(compConfigs.size(), 1ul) << "One default config expected.";
+  EXPECT_EQ(compConfigs.at(0).state, scr::dto::ComponentState::ACTIVE)
+    << "Ignore policy component state should be ACTIVE";
+
+  auto configAdminService =
+    GetInstance<cppmicroservices::service::cm::ConfigurationAdmin>();
+  ASSERT_TRUE(configAdminService)
+    << "GetService failed for ConfigurationAdmin.";
+
+  // Create configuration object, try to update property
+  // and property update should be ignored.
+  auto configuration = configAdminService->GetConfiguration(componentName);
+  auto configInstance = configuration->GetPid();
+
+  // Confirm configuratino object presented and check component state.
+  compConfigs = GetComponentConfigs(testBundle, componentName, compDescDTO);
+  EXPECT_EQ(compConfigs.size(), 1ul) << "One default config expected.";
+  EXPECT_EQ(compConfigs.at(0).state, scr::dto::ComponentState::ACTIVE)
+    << "ACTIVE is expected since configuration object is created.";
+
+  // GetService to make component active.
+  auto instance = GetInstance<test::CAInterface>();
+  ASSERT_TRUE(instance) << "GetService failed for CAInterface";
+
+  cppmicroservices::AnyMap props(
+    cppmicroservices::AnyMap::UNORDERED_MAP_CASEINSENSITIVE_KEYS);
+  const std::string instanceId{ "instance1" };
+  props["uniqueProp"] = instanceId;
+  configuration->Update(props);
+
+  // Confirm component instance was created with the properties has default value.
+  EXPECT_TRUE(instance->GetProperties().empty())
+    << "Property instance should only have default value.";
+} // end of testWithoutPID
+
+/**
+* Test the component instance is created with default properties when only the 
+* configuration PID is specified (no policy specified).
+*/
+TEST_F(tServiceComponent, testWithoutConfigPolicy) // DS_CAI_FTC_15
+{
+  // Start the test bundle containing the component name.
+  std::string componentName = "sample::ServiceComponentCA15";
+  cppmicroservices::Bundle testBundle = StartTestBundle("TestBundleDSCA15");
+
+  // Use DS runtime service to validate the component state.
+  scr::dto::ComponentDescriptionDTO compDescDTO;
+  auto compConfigs =
+    GetComponentConfigs(testBundle, componentName, compDescDTO);
+
+  EXPECT_EQ(compConfigs.size(), 1ul) << "One default config expected.";
+  EXPECT_EQ(compConfigs.at(0).state, scr::dto::ComponentState::ACTIVE)
+    << "Ignore policy component state should be ACTIVE";
+
+  auto configAdminService =
+    GetInstance<cppmicroservices::service::cm::ConfigurationAdmin>();
+  ASSERT_TRUE(configAdminService)
+    << "GetService failed for ConfigurationAdmin.";
+
+  // Create configuration object, try to update property
```

Comment:
Should move all of these lines to after the GetInstance on line 122. What we want this test to confirm is that DS does not call the component instance Modified method when configuration-policy is "ignore". To confirm that you have to get the component instance first and then update the configuration object.